### ISSUE TITLE
documentation contribution

### ISSUE THREAD

**Issue by ericmandel** [ericmandel/pyds9#36]
@montefra Jane Rigby (GSFC) contributed a short tutorial on using pyds9, which look like this:

<img width="1097" alt="pyds9 ipython screenshot" src="https://cloud.githubusercontent.com/assets/3028875/16454359/78fdd9a6-3dde-11e6-90f2-dfcceadb71b3.png">

Do you have a preferred way to add this to our documentation, or otherwise make it easily accessible to you for evaluation?  I can't add the the ipynb file here.

**Comment by ericmandel** [comment]
@montefra Indeed, Jane as added more tutorials to her GitHub account:

https://github.com/janerigby/Tutorials

What is the best way to handle these?

**Comment by montefra** [comment]
@ericmandel : if we create a sphinx documentation (and we probably should), we can add the notebooks as shown [here](http://sphinx-ipynb.readthedocs.io/en/latest/howto.html). Of course if @janerigby agrees

**Comment by ericmandel** [comment]
@montefra Yes, Jane says that she is happy to contribute ... and a sphinx documentation project would be good. My original pyds9 Makefile has an "html" entry that does some (probably old-style?) sphinx conversion, so there is a start. I can't do anything about this now, and I'm sure you have a few things to do as well. So either one of us will get to it, or perhaps someone else can volunteer.

**Comment by montefra** [comment]
astropy_helpers has something to build sphinx. I can take care of it in the devel branch. I thing that I'm done with #28

**Comment by ericmandel** [comment]
OK, great ... and great work in general!
